docs/widgets.md

```diff
@@ -248,6 +248,63 @@ Note the addition required value in the entity file:
 
 The [rssToJson][] microservice is a fine way to convert desired RSS feeds into the required JSON representation.
 
+### Action Items List
+
+![action items widget](./img/action-items.png)
+
+```xml
+<name>widgetType</name>
+<value>action-items</value>
+```
+
+#### When to use
+
+* You want to display a list of quantity-based items, with quantities that are expected to change. For example, a manager who has to approve time off could see "5 leave requests need your approval".
+
+#### Additional entity file configuration
+
+```xml
+<portlet-preference>
+    <name>widgetType</name>
+    <value>action-items</value>
+</portlet-preference>
+<portlet-preference>
+    <name>widgetConfig</name>
+    <value>
+      <![CDATA[{
+        "actionItems": [
```

Comment:
Someday the neighborhood of `portlet-definition`s will evolve, either such that `portlet-definition` entity files can be represented as something other than XML or such that we're not using `portlet-definition`s as the substrate for representing widget configuration.  In the meantime, widgets ride along as magic portlet preferences in `portlet-definition` entity files.